app_dart/lib/src/service/buildbucket.dart

```diff
@@ -52,8 +61,9 @@ class BuildBucketClient {
   Future<T> _postRequest<S extends JsonBody, T>(
     String path,
     S request,
-    T Function(Map<String, dynamic>? rawResponse) responseFromJson,
-  ) async {
+    T Function(Map<String, dynamic>? rawResponse) responseFromJson, {
+    String buildBucketUri = kDefaultBuildBucketBuildUri,
```

Comment:
Since these are injectable in the constructor, do we need to pass it as a parameter as well?

Reply:
Yeah, we need this parameter to distinguish between build and builder requests